addons/backgrounds/README.md

```diff
@@ -28,19 +30,23 @@ Then write your stories like this:
 
 ```js
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 
-storiesOf('Button', module)
-  .addParameters({
+export default {
+  title: 'Button',
+  parameters: {
     backgrounds: [
       { name: 'twitter', value: '#00aced', default: true },
       { name: 'facebook', value: '#3b5998' },
-    ],
-  })
-  .add('with text', () => <button>Click me</button>);
+    ]
+  },
+};
+
+export const defaultView = () => (
```

Comment:
Looks like maybe the names got messed up in the rewrite? `withText`?

Reply:
If a storyFile only has 1 variant in it. I figured I'd just make make it `defaultView` for simplicity.

Reply:
WFM

Reply:
`withtext` might mean something to someone https://github.com/storybookjs/storybook/pull/8962#pullrequestreview-323662686